editor/tests/org.fusesource.ide.wsdl2rest.ui.tests.integration/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <groupId>org.fusesource.ide.editor</groupId>
+    <artifactId>tests</artifactId>
+    <version>10.3.0-SNAPSHOT</version>
```

Comment:
as version has been upgraded will need to modify to 11.0.0 (and rebase on master)